e2e/cypress/integration/collapsed_reply_threads/following_spec.js

```diff
@@ -222,6 +231,12 @@ describe('Collapsed Reply Threads', () => {
                 cy.findByText('F').should('be.visible');
             });
         });
+
+        // # Click Unreads button
+        cy.findByText('Unreads').click();
+
+        // # Search guidance item should not be shown at the end of the Unreads threads list
+        cy.get('.ThreadList .no-results__wrapper').should('not.exist');
```

Comment:
🚀 Nice test!

Nit: I think the rule for the comment is that an assertion comment must start with `// *` instead of `// #`